### ISSUE TITLE
Spurious Route53 records added to plan after upgrade to 0.7.10

### ISSUE THREAD

**Issue by onetwopunch** [hashicorp/terraform#10398]
Since we upgraded to 0.7.10 we've been getting some spurious route53 records (1 of 7 shown below)  appearing in our plan, but not actually changing anything in the state.

## Version

0.7.13

## Plan Output

```
~ module.cluster.aws_route53_record.platform
    alias.2629901491.evaluate_target_health: "false" => "false"
    alias.2629901491.name:                   "staging-platform-elb.us-west-1.elb.amazonaws.com." => ""
    alias.2629901491.zone_id:                "Z368ELLRRE2KJ0" => ""
    alias.3839681763.evaluate_target_health: "" => "false"
    alias.3839681763.name:                   "" => "staging-platform-elb.us-west-1.elb.amazonaws.com"
    alias.3839681763.zone_id:                "" => "Z1M58G0W56PQJA"
```

## Terraform code

``` hcl
resource "aws_route53_record" "platform" {
  zone_id = "${var.route53_zone_id}"
  name = "${var.platform_host}"
  type = "A"

  alias {
    name = "${aws_elb.platform.dns_name}"
    zone_id = "${aws_elb.platform.zone_id}"
    evaluate_target_health = false
  }
}
```

The reason I'm asking this as an issue instead of a StackOverflow is because it failed in correlation with a version change, so maybe there's some breaking change in the API.

**Comment by sheeley** [comment]
likely the same as #10004

**Comment by stack72** [comment]
Hi @onetwopunch 
Apologies for the issue here - I am actively working on this issue as it was due to a *fix* applied to refresh route53 alias's in 0.7.10

Please stay tuned for more deets

Paul

**Comment by onetwopunch** [comment]
@stack72 Thanks for the quick response, glad it's not on our end, I'll keep watching. Let me know if you need a hand. 👍
